### PR TITLE
Integration & unit tests for examples

### PR DESCRIPTION
**Pull request by chrisbennight**
* Adds geowave-examples to the pom build matrix
  * Adds unit tests for the geowave-examples instances
  * Adds integration test using the geowave-examples data set
    * Validates WMS output
    * Rendering differences between 2.5 and 2.6 cause different outputs for the same basic grid

### CODE REVIEW

File: geowave-test/src/main/java/mil/nga/giat/geowave/test/service/ServicesTestEnvironment.java [hunk 1]
```diff
@@ -96,6 +103,7 @@ public static void startServices() {
 							":",
 							";"));
 					gsWebapp.setClassLoader(classLoader);
+					gsWebapp.setParentLoaderPriority(true);
```
Comment (chrisbennight) [inline geowave-test/src/main/java/mil/nga/giat/geowave/test/service/ServicesTestEnvironment.java]: Ran into imageio related sealed class exceptions without this; setting this to true creates the same behaviour as tomcat.   
@spohnan should check this for our standalone version (can also be set in the WEB-INF settings file as well for jetty)

**Comment by chrisbennight** [comment]
#278

**Comment by jwomeara** [comment]
This is where I say, "looks good to me," right?

**Comment by rfecher** [comment]
"looks good to me" (too late Whitney)

**Comment by jwomeara** [comment]
Hey, what does this "revert" button do?

**Pull request merged**